### ISSUE TITLE
Add support for Amnezia WireGuard

### ISSUE THREAD

**Issue by mhz-tamb** [subs1stem/wg-assistant#6]
It would be great if the bot could support managing configurations for Amnezia WireGuard.

The main change would involve adding support for additional configuration fields when generating peers, as described in the [Amnezia WireGuard documentation](https://github.com/amnezia-vpn/amneziawg-linux-kernel-module?tab=readme-ov-file#configuration).

These additional fields should be identical for both client and server configurations, except for the Jc field, which is specific to the client side.

**Comment by subs1stem** [comment]
Yes, it may be useful for people from some countries where the Internet is strictly regulated by the government. I will do it.

**Comment by subs1stem** [comment]
Implemented in [v1.5.0](https://github.com/subs1stem/wg-assistant/releases/tag/v1.5.0)
